README.md

```diff
@@ -140,6 +141,10 @@ and [stunnel](http://linux.die.net/man/8/stunnel) configurations to see what set
 - `PGBOUNCER_LOG_POOLER_ERRORS` Default is yes
 - `PGBOUNCER_STATS_PERIOD` Default is 60
 - `PGBOUNCER_SERVER_RESET_QUERY` Default is empty when pool mode is transaction, and "DISCARD ALL;" when session.
+- `PGBOUNCER_QUERY_TIMEOUT` Default is 0.0 (disabled)
+- `PGBOUNCER_QUERY_WAIT_TIMEOUT` Default is 0.0 (disabled)
+- `PGBOUNCER_CLIENT_IDLE_TIMEOUT` Default is 0.0 (disabled)
+- `PGBOUNCER_IDLE_TRANSACTION_TIMEOUT` Default is 0.0 (disabled)
```

Comment:
I don't think these settings are proper to make available to this buildpack, as in my experience they lead to application db pools with no usable connections and deadlocked dynos. They are listed as `Dangerous timeouts` in https://pgbouncer.github.io/config.html and while the feature description is alluring, the mechanism by which these timeouts are implemented is unsafe for most apps.